.github/config/regression-tests.yml

```diff
@@ -0,0 +1,805 @@
+
+      # unknown cheatcode with selector 0xdd9fca12
+      test/base/BaseReactor.t.sol
+      test/reactors/PriorityOrderReactor.t.sol
+      test/reactors/V3DutchOrderReactor.t.sol
+      test/reactors/V2DutchOrderReactor.t.sol
+      test/reactors/LimitOrderReactor.t.sol
+      test/reactors/BaseDutchOrderReactor.t.sol
+      test/reactors/ExclusiveDutchOrderReactor.t.sol
+      test/reactors/DutchOrderReactor.t.sol
+    ref: 4013dfa4bc53b823b406b035a9b5eb579607eb99
+  Vectorized/solady:
+    forge-version: v0.3.0
+    hardhat-config: |
+      export default {
+        "paths": {
+          "sources": "src",
+          "tests": {
+            "solidity": "test"
+          }
+        },
+        "solidity": {
+          "version": "0.8.28",
+          "remappings": [
+            "forge-std=test/utils/forge-std/"
+          ],
+          "settings": {
+            "optimizer": {
+              "enabled": true,
+              "runs": 1000
+            },
+            "evmVersion": "cancun"
+          }
+        },
+        "solidityTest": {
+          "blockGasLimit": BigInt(100000000),
+          "fsPermissions": {
+            "read": [
+              "./test/data"
+            ]
+          },
+          "fuzz": {
+            "runs": 256
+          },
+          "invariant": {
+            "depth": 15,
+            "runs": 10
+          }
+        }
+      };
+    ignore: |
+      */*7702*
+      */*Transient*
+      */ext/ithaca/*
+      */ext/zksync/*
+
+      # testP256Normalized(uint256,bytes32): unknown cheatcode with selector 0xc453949e
+      test/P256.t.sol
+      test/WebAuthn.t.sol
+
+      # testBumpSlot(bytes32,uint256)
+      test/LibStorage.t.sol
+
+      # testTargetGenerate(): EvmError: Revert
+      test/DeploylessPredeployQueryer.t.sol
+    ref: c9e079c0ca836dcc52777a1fa7227ef28e3537b3
+  foundry-rs/forge-std:
+    forge-version: v0.3.0
+    hardhat-config: |
+      export default {
+        "paths": {
+          "sources": "src",
+          "tests": {
+            "solidity": "test"
+          }
+        },
+        "solidity": {
+          "version": "0.8.26",
+          "settings": {
+            "optimizer": {
+              "enabled": true,
+              "runs": 200
+            }
+          }
+        },
+        "solidityTest": {
+          "fsPermissions": {
+            "readWrite": [
+              "./"
+            ]
+          },
+          "rpcEndpoints": {
+            "mainnet": "https://eth-mainnet.alchemyapi.io/v2/WV407BEiBmjNJfKo9Uo_55u0z0ITyCOX",
+            "optimism_sepolia": "https://sepolia.optimism.io/",
+            "arbitrum_one_sepolia": "https://sepolia-rollup.arbitrum.io/rpc/",
+            "needs_undefined_env_var": "${UNDEFINED_RPC_URL_PLACEHOLDER}"
+          }
+        }
+      };
+    ignore: |
+      # test_DeriveRememberKey(): unknown cheatcode with selector 0x6229498b; you may have a mismatch between the `Vm` interface (likely in `forge-std`) and the `edr_solidity_tests` version
```

Comment:
`deriveKey(string,uint32)` cheatcode which is deployment related and we're not planning to support it

Reply:
```suggestion
      # test_DeriveRememberKey(): unknown cheatcode with selector 0x6229498b (deriveKey(string,uint32))
```